.travis.yml

```diff
@@ -1,32 +1,26 @@
-sudo: false
+dist: bionic
```

Comment:
I strongly recommend trying the CircleCI setup. It builds really fast, and all Mopidy projects share a common build image that we can improve without having to fix things across all the extensions.

The steps needed are roughly:

1. Copy the `.circleci` dir from the cookiecutter.
2. Remove the `{% raw %}` and `{% endraw %}` markers in the cookiecutter file.
3. Find the project at https://circleci.com/gh/organizations/mopidy/settings#projects
4. Open Project settings on the right project
5. Click the "Follow project" button
6. Open Project settings > Advanced settings
7. Set "Build forked pull requests" to "On"
8. CircleCI is now ready to build any branch/PR with a `.circleci/config.yml`
9. Setup of coverage data from CircleCI to Codecov.io is automatic

Reply:
I took a quick look at CircleCI, but I refrained from logging into the service, since a login is only possible with a github account (or bitbucket), without being able to restrict control over the granted read access (e.g. my email address and unrelated projects).

But if you want to adjust the above settings in CircleCI, then I will be happy to add the `.circleci/config.yml` file.

Reply:
I've turned on CircleCI for this repo now: https://circleci.com/gh/mopidy/mopidy-beets

That is, I've done step 3-7 in my previous comment.